hawc/apps/assessment/models.py

```diff
@@ -91,7 +91,11 @@ def get_dashboard_url(self) -> str:
         return f"https://comptox.epa.gov/dashboard/dsstoxdb/results?search={self.dtxsid}"
 
     def get_img_url(self) -> str:
-        return f"https://comptox.epa.gov/dashboard-api/ccdapp1/chemical-files/image/by-dtxsid/{self.dtxsid}"
+        # TODO - always use api-ccte.epa.gov when API key is no longer required
```

Comment:
I made one more change; for now, it falls-through to the old API without a key since that still exists. Hopefully in the future we'll be able to remove this...